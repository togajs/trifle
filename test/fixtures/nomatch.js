import Foo from 'foo';

export default class Bar extends Foo {
	constructor(options) {
		super(options);
	}

	log(value) {
		console.log(value);
		return this;
	}
}
