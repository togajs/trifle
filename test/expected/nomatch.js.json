{
  "type": "Documentation",
  "body": [
    {
      "type": "CommentBlock",
      "description": "",
      "trailingCode": "import Foo from 'foo';\n\nexport default class Bar extends Foo {\n\tconstructor(options) {\n\t\tsuper(options);\n\t}\n\n\tlog(value) {\n\t\tconsole.log(value);\n\t\treturn this;\n\t}\n}",
      "tags": []
    }
  ]
}
